examples/fastrtc_ui.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+## Setup
```

Comment:
Do you want to put your name on this somewhere?

Mainly because you made it, you should get the credit, but it also helps readers understand that this might not be a direct/official Google sample and sets some expectations (e.g. if we change an API or model and don't update this sample immediately)

Reply:
Yes great point - done!